src/main/java/org/codeforamerica/messaging/config/WebConfig.java

```diff
@@ -16,8 +18,20 @@ public class WebConfig implements WebMvcConfigurer {
     @Value("${server.trustedPathPrefix:null}")
     private String trustedPathPrefix;
 
+    @Value("${twilio.auth.token}")
+    private String twilioAuthToken;
+
     @Bean
     public FilterRegistrationBean<TrustedEndpointsFilter> trustedEndpointsFilter() {
         return new FilterRegistrationBean<>(new TrustedEndpointsFilter(trustedPort, trustedPathPrefix));
     }
+
+    @Bean
+    @Profile("!test && !ci")
```

Comment:
@vrajmohan When this is removed, then `TwilioCallbackControllerTest` will fail. The Twilio package provides a `RequestValidator` that is a big challenge to mock out for testing, so I opted to remove this filter for tests. Manual tests in staging show that previously I could spoof a twilio callback and change the status of a message, but with this change I get "401 Unauthorized".